battleship.rb

```diff
@@ -0,0 +1,84 @@
+# require 'pry'
+# attr_accessor :radar, :ocean
+#Generates the board, called ocean
+x_axis = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j']
+y_axis = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']
+@ocean = x_axis.product(y_axis).each do |combo|
```

Comment:
When I didn't have them the code triggered a failure of "no implicit conversion from nil to to integer".(it is on line 34, so I suspect there is a ripple that I can't follow fully) The instance variable gets me to the point that I can call an attack (so long as I don't hit the ship, that triggers the same error on line 69. That's really where I am: bouncing between about three errors. I'm going to keep banging on it, but I also think I need to get  more fluency with ruby so that I can follow the errors to the root better.